### PR TITLE
[WIP] Feat: remove update-notifier-common for Ubuntu

### PR DESCRIPTION
**Pull request by mbologna**
To be tested.

### CODE REVIEW

**Comment by Bischoff** [comment]
Currently testing. Missing dash before pkg.removed: ?

File: salt/minion/testsuite.sls [hunk 1]
```diff
@@ -22,6 +22,14 @@ minion_cucumber_requisites:
     - require:
       - sls: default
 
+{% if grains['os_family'] == 'Debian' %}
```
Comment (moio) [inline salt/minion/testsuite.sls]: Please avoid nested/indented blocks if possible (YAML is picky on whitespace). In this case I would suggest a simple `and`.

Thanks!

**Comment by Bischoff** [comment]
Retested, works (in the sense it removes updates-notifier-common package). Merging.

**Comment by Bischoff** [comment]
No effect, I suggest reverting. See however #6267 where I might know what's happening.

**Comment by moio** [comment]
Can you please revert this @mbologna? Feel free to just commit without a PR in this specific case.

**Comment by mbologna** [comment]
> Can you please revert this @mbologna? Feel free to just commit without a PR in this specific case.

Done in https://github.com/moio/sumaform/pull/499

**Pull request merged**